src/routing_table/mod.rs

```diff
@@ -1129,7 +1129,8 @@ mod tests {
         assert_eq!(our_name, table.our_name);
 
         // Get some names
-        let close_name: u32 = *unwrap!(table.our_section.iter().nth(4));
+        let close_name: u32 =
+            *unwrap!(table.our_section.iter().filter(|name| **name != our_name).nth(4));
```

Comment:
Ultimately I wanted to see if we can let `table.iter()` include our name, but this suffices for now I guess.